poetry.lock

```diff
@@ -421,15 +189,13 @@ PyQt5-sip = ">=12.11,<13"
 
 [[package]]
 name = "pyqt5-qt5"
-version = "5.15.2"
+version = "5.15.13"
```

Comment:
Something went wrong here, not sure why this package was upgraded. Let me back it out and see if that fixes the CI issue.

Reply:
Ah yeah, that'll do it; it tries to update everything unless you use `--no-update`